src/resolvers-cassandra/Settings/queries.js

```diff
@@ -46,12 +46,40 @@ function sites(args, res) { // eslint-disable-line no-unused-vars
 }
 
 /**
+ * @param {*} row 
+ * @returns { accountName: string, consumerKey: string, consumerSecret: string, token: string, tokenSecret: string }
+ */
+function cassandraRowToTwitterAccount(row) {
+  return {
+    accountName: row.params.accountName,
+    consumerKey: row.params.consumerKey,
+    consumerSecret: row.params.consumerSecret,
+    token: row.params.token,
+    tokenSecret: row.params.tokenSecret
+  };
+}
+
+/**
  * @param {{siteId: string}} args
  * @returns {Promise.<{runTime: string, accounts: Array<{accountName: string, consumerKey: string, consumerSecret: string, token: string, tokenSecret: string}>}>}
  */
 function twitterAccounts(args, res) { // eslint-disable-line no-unused-vars
+  return new Promise((resolve, reject) => {
+    const sourcesByConnector = 'SELECT * FROM fortis.streams WHERE connector = ? ALLOW FILTERING';
```

Comment:
Sounds good. The alternative would be to add an index but it's likely better to keep our memory free for more needed indices.